src/mixs/schema/ancient.yml

```diff
@@ -289,9 +288,10 @@ slots:
       - sequencing
     slot_uri: MIXS:999999912
     range: integer
+    multivalued: true
   num_reamp_cycles:
     description: >-
-      Number of amplification cycles after library indexing PCR
+      Number of amplification cycles after library indexing PCR. This refers to amplifications prior capture experiments.
```

Comment:
This doesn't have to be associated with a capture so if anything it probably should be "This refers to the reamplification of libraries following indexing PCR and prior to additional experimental work such as target enrichment"